CHANGES.md

```diff
@@ -1584,22 +1584,22 @@ Memory Improvements
 Documentation
 -------------
 * dramatically improved the organization of the online Chapel documentation
-  (see http://chapel.cray.com/docs/1.13/)
+  (see https://chapel-lang.org/docs/1.13/)
 * created web-based versions of the 'chpl' and 'chpldoc' man pages
-  (see http://chapel.cray.com/docs/1.13/usingchapel/man.html
-   and http://chapel.cray.com/docs/1.13/tools/chpldoc/man.html)
+  (see https://chapel-lang.org/docs/1.13/usingchapel/man.html
+   and https://chapel-lang.org/docs/1.13/tools/chpldoc/man.html)
 * linked to the language specification and quick reference from the online docs
-  (see http://chapel.cray.com/docs/1.13/language/spec.html
-   and http://chapel.cray.com/docs/1.13/language/reference.html)
+  (see https://chapel-lang.org/docs/1.13/language/spec.html
+   and https://chapel-lang.org/docs/1.13/language/reference.html)
 * converted a number of top-level README-style files to markdown format
   (e.g., see $CHPL_HOME/CHANGES.md, CONTRIBUTORS.md, PERFORMANCE.md, ...)
-* moved the Chapel evolution page from chapel.cray.com to the online docs
-  (see http://chapel.cray.com/docs/1.13/language/evolution.html)
+* moved the Chapel evolution page from chapel-lang.org to the online docs
```

Comment:
missing https:// (probably not important, I just have a script that was going to scrape the urls and it tripped over this since it was missing the https://)